src/FiniteVolumeTools.cpp

```diff
@@ -244,7 +244,7 @@ void GetTriangleThatContainsPoint(
 	}
 	
 	// First check if point is in initial face
-	if (fTriangleContainsPoint(mesh,iFaceInitial,dX,dY)){
+	if (DoesTriangleContainPoint(mesh,iFaceInitial,dX,dY)){
```

Comment:
Is it guaranteed that mesh contains only triangles?  Why use DoesTriangleContainPoint and not DoesFaceContainPoint?

Reply:
Yes, this function is only called for the Delaunay triangulation remapping, where mesh is guaranteed to be triangular.